### PR TITLE
Dispersion quick method API Update

### PR DESCRIPTION
**Pull request by mgarod**
I have made the following changes:

1. Fix some typos
2. Add documentation for the quick method and at-a-glance box.
3. Reorder DispersionPlot and quick method parameters and docstrings so that they are in sync.
4. Change quick method `words` to `target_words` which brings it in line with the DispersionPlot parameter. Made same change in the test.
5. Add show parameter to quick method.

Not much else to add!

### CODE REVIEW

File: docs/api/text/index.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Text Modeling Visualizers
 
 Yellowbrick provides the ``yellowbrick.text`` module for text-specific visualizers. The ``TextVisualizer`` class specifically deals with datasets that are corpora and not simple numeric arrays or DataFrames, providing utilities for analyzing word dispersion and distribution, showing document similarity, or simply wrapping some of the other standard visualizers with text-specific display properties.
 
-We currently have four text-specific visualizations implemented:
+We currently have five text-specific visualizations implemented:
```
Comment (pdamodaran) [inline docs/api/text/index.rst]: Thanks for making this update!

File: tests/test_text/test_dispersion.py [hunk 2]
```diff
@@ -40,14 +40,14 @@
 class TestDispersionPlot(VisualTestCase):
     def test_quick_method(self):
         """
-        Assert no errors occur when using the qucik method
+        Assert no errors occur when using the quick method
```
Comment (pdamodaran) [inline tests/test_text/test_dispersion.py]: Thanks for fixing the typo.

**Comment by bbengfort** [comment]
@mgarod @pdamodaran the hold up here is still AppVeyor -- I'm a bit mystified by what's going on here. Rerunning the tests to see if they fixed it on their end.

**Comment by pdamodaran** [comment]
@mgarod - thanks for tracking down the issue with the AppVeyor! I will go ahead and merge this in!

**Pull request merged**